apps/web/src/components/widget/AvatarContainer.tsx

```diff
@@ -0,0 +1,224 @@
+import { useState, Dispatch, SetStateAction } from 'react';
+import { Box, Group, Divider, Popover, Stack, Avatar as MAvatar } from '@mantine/core';
+import {
+  WarningFilled,
+  InfoCircleFilled,
+  CheckCircleFilled,
+  CloseCircleFilled,
+  UpCircleFilled,
+  QuestionCircleFilled,
+} from '@ant-design/icons';
+import { useController } from 'react-hook-form';
+import { SystemAvatarIconEnum, IAvatarDetails, AvatarTypeEnum } from '@novu/shared';
+import { colors, Input, Switch, Text, Tooltip } from '../../design-system';
+import { Camera } from '../../design-system/icons/general/Camera';
+import { Avatar } from '../../design-system/icons/general/Avatar';
```

Comment:
You can export both icons from `../../design-system/icons/index.ts` so they can share the import place like the rest of the icons used in the design system.